src/containers/community/CommunityEditor.js

```diff
@@ -313,7 +316,7 @@ export default class CommunityEditor extends React.Component {
           {errors.server || 'The information you provided has some errors; please see above.'}
         </p>}
         <button type='button' onClick={this.submit} disabled={disableSubmit}>
-          {validating || saving ? 'Please wait...' : 'Save'}
+          {validating || saving || uploadingImage ? 'Please wait...' : 'Save'}
```

Comment:
is there any reason this line is not just
`{disableSubmit ? 'Please wait...' : 'Save'}`